specs/protocol/system-config.md

```diff
@@ -245,13 +270,15 @@ Returns the block number at which the op-node can start searching for logs.
 
 ### paused
 
-This function integrates with the [Pause Mechanism](./stage-1.md#pause-mechanism) by using the
-chain's `ETHLockbox` address as the [Pause Identifier](./stage-1.md#pause-identifier). Returns the
-current pause state of the system by checking if the `SuperchainConfig` is paused for this chain's
-`ETHLockbox`.
+This function integrates with the [Pause Mechanism](./stage-1.md#pause-mechanism) by using either the
```

Comment:
the Pause Mechanism spec in that document should be updated to allow the `OptimismPortal` address to be used as an identifier. It currently says only the lock box and 0 are valid identifiers.